packages/backend/src/controllers/renameController.ts

```diff
@@ -95,7 +95,7 @@ export class RenameController extends BaseController {
 
         return {
             status: 'ok',
-            results: undefined,
+            results: { jobId },
```

Comment:
now we return a jobId, so we can wait until it is finishe in the UI and report the number of updated charts